www/main/settings/components/Setting.jsx

```diff
@@ -131,6 +144,17 @@ const Setting = ({ settingObj, stateVar, stateFunction }) => {
         </>
       );
       break;
+    case 'reset-button':
+      settingDOM = (
+        <div
```

Comment:
@yadvirkaur Can we use a button tag here, I think that fits more nicely with the setting type.